src/main/java/com/ibm/cloud/sdk/core/util/MapValueObjectTypeAdapter.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright (C) 2011 Google Inc.
```

Comment:
Should we still have a Google copyright?

Reply:
You're right, I should add our own at the top.   But perhaps we should also include the original?   I essentially copied a class from GSON and then made some minor changes to it.